props/prop_base.h

```diff
@@ -527,6 +527,10 @@ class PropBase : CommandParser, Looper, protected SaberBase, public ModeInterfac
   }
 
   virtual void SpeakBladeID(float id) {
+    if (&SFX_mnum) {
```

Comment:
Why?
Speak blade id is a debug function, it doesn't need to sound good.
Also, it doesn't  compile, because the { } are unbalanced.